src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -107,14 +150,42 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
             return new Subscription(invocationHandler, invocationList);
         }
 
+        /// <summary>
+        /// Invokes a streaming hub method using the specified method name, return type and arguments.
```

Comment:
Invokes a streaming hub method **on the server** using the specified method name, return type and arguments.